hi-ml-histopathology/src/histopathology/utils/output_utils.py

```diff
@@ -147,29 +143,30 @@ def save_features(results: ResultsType, outputs_dir: Path) -> None:
     torch.save(features_list, outputs_dir / 'test_encoded_features.pickle')
 
 
-def save_top_and_bottom_tiles(results: ResultsType, n_classes: int, figures_dir: Path) \
+def save_top_and_bottom_tiles(results: ResultsType, n_classes: int, figures_dir: Path, k_tiles: int = 10) \
```

Comment:
as mentioned on other places, I find `k_tiles` confusing, what is `k`? Rename to `num_tiles`?